cli/client/main.go

```diff
@@ -416,14 +413,33 @@ func mainWithStatusCode() int {
 	}
 
 	hostnameIsAnIP := net.ParseIP(hostname) != nil
-
-	port := configPort
-	if port == -1 && urlPort != "" {
-		port, err = strconv.Atoi(urlPort)
-		if err != nil {
-			log.Error().Msgf("invalid port number: %s: %s", urlPort, err)
+	// *** Port selection ***
```

Comment:
Overall I am not a huge fan of the table. I think we can make it more concise. The error case when the port cannot be parsed seems obvious.
I think it is fine if we just say that the CLI port takes precedence over the config port when specified.